export/tasks_generator.go

```diff
@@ -105,12 +122,19 @@ func (exportID taskIDGenerator) finishedTaskID() parade.TaskID {
 	return parade.TaskID(fmt.Sprintf("%s:finished", exportID))
 }
 
-func (exportID taskIDGenerator) copyTaskID(physicalAddress string) parade.TaskID {
-	return parade.TaskID(fmt.Sprintf("%s:copy:%s", exportID, physicalAddress))
+func (exportID taskIDGenerator) startedTaskID() parade.TaskID {
+	return parade.TaskID(fmt.Sprintf("%s:started", exportID))
```

Comment:
This is a simple past-tense verb, whereas in other task IDs we use an imperative verb.  Could you do that here, too?  (Not sure which imperative form is intended here!)

Reply:
👌